eng/tools/lint-diff/package.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "@azure-tools/lint-diff",
+  "private": true,
+  "type": "module",
+  "main": "dist/src/lint-diff.js",
+  "bin": {
+    "lint-diff": "cmd/lint-diff.js"
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "test": "vitest",
+    "test:ci": "vitest run --coverage --reporter=verbose",
+    "clean": "rm -rf dist && rm -rf node_modules"
+  },
+  "engines": {
+    "node": ">= 18.0.0"
```

```suggestion
    "node": ">= 20.0.0"
```
Min node in specs repo recently bumped to 20